bindata/network/sriov/sriov-device-plugin.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: sriov-device-plugin
+  namespace: sriov-device-plugin
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the SR-IOV Network Device Plugin.
+spec:
+  selector:
+    matchLabels:
+      app: sriovdp
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: sriovdp
+        component: network
+        type: infra
+        openshift.io/component: network
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/os: linux
```

Comment:
Sriov-device-plugin fails to run on nodes which doesn't contain SR-IOV capable devices or VFs not created.
Perhaps worth a note here to indicating further improvement is needed to run SR-IOV device plugin on selected nodes. This might be solved later with NFD to add labels to SR-IOV capable nodes.